safedelete/models.py

```diff
@@ -200,6 +210,14 @@ def _perform_unique_checks(self, unique_checks):
                 )
         return errors
 
+    @property
+    def safedelete_policy(self):
```

Comment:
Underscores in Python indicate that it's a private field. So adding a property to make it public kind of defeats the purpose.

Reply:
Yeah, you are right. I needed a setter in the first verison. I'll remove it.